app/controllers/static_pages_controller.rb

```diff
@@ -0,0 +1,14 @@
+class StaticPagesController < ApplicationController
+  def home
+  end
```

Comment:
method trống phần thân thì viết trên 1 line thôi em nhé
``dè home; end`
chekc lại toàn bộ